android/expoview/src/main/java/host/exp/exponent/Constants.java

```diff
@@ -65,6 +65,7 @@ public static void setSdkVersions(List<String> sdkVersions) {
 
   static {
     Set<String> abiVersions = new HashSet<>();
+    abiVersions.add("UNVERSIONED");
```

Comment:
remove, i think this was only for getting the published test-suite to run locally